bool goalie_attacking
float32 time_to_reach_ball
float32 time_to_reach_striker
float32 time_to_reach_midfielder
float32 time_to_reach_defender
uint8 current_role

uint8 ROLE_NONE        = 0
uint8 ROLE_GOALIE      = 1
uint8 ROLE_STRIKER     = 2
uint8 ROLE_DEFENDER    = 3
uint8 ROLE_MIDFIELDER  = 4
